src/components/Book.js

```diff
@@ -14,7 +14,18 @@ function Book({ bookData }) {
           <h4 className={styles.bookCat}>{bookData.author}</h4>
           <ul className={styles.buttonList}>
             <li>Comments</li>
-            <li><button type="button" onClick={() => dispatch(remove({ bookData }))}>Remove</button></li>
+            <li>
+              <button
+                type="button"
+                onClick={async () => {
+                  await dispatch(removeBook(bookData.item_id));
+                  await dispatch(getBooksFromApi());
+                }}
+              >
+                Remove
+              </button>
+
+            </li>
```

Comment:
- [ ] Great job on connecting your bookstore to an API as specified in the project requirement! 👏 However, there is a minor UI issue 🐞 that we need to address before moving on to the next milestone.
- [ ] While testing your app I realized that on page load, your home page looks blank with just the form input. Kindly make sure that on page load all books in your API are rendered on the home page as shown in the original design. See below for more details:
<table>
<tr>
<th align=“middle”>On page load</th>
<th align=“middle”>After adding a book</th>
</tr>
<tr>
<td>
<img width="1439" alt="Screenshot 2023-07-19 at 17 32 35" src="https://github.com/Kidd254/Bookstore-React-Redux/assets/107192076/fe4fef2b-e666-4eb5-950f-f581d67ef9f5">
</td>
<td><img width="1439" alt="Screenshot 2023-07-19 at 17 32 55" src="https://github.com/Kidd254/Bookstore-React-Redux/assets/107192076/501a7292-ce91-4ba2-9747-c4cd47ac725f"></td>
</tr>
</table>

- [ `OPTIONAL`] Your design doesn't follow the layout of the original design given in the project requirements so kindly make sure it does